scripts/build-v2.py

```diff
@@ -101,7 +101,7 @@ def condenseFont(font, scale=.8, stemWidth=185):
 proj.generateFont(Mix([rg, bd], RPoint(0.73, 0.73)),
                   "%s/Bold/Bold/Rg"%FAMILYNAME)
 proj.generateFont(Mix([rg, bd], RPoint(1.125, 1.0)),
-                  "%s/Black/Bold/Bk"%FAMILYNAME)
+                  "%s/Black/Regular/Bk"%FAMILYNAME)
```

Comment:
This is the way we designate instance metadata; these strings are handled in [instanceNames.py](https://github.com/google/roboto/blob/master/scripts/lib/fontbuild/instanceNames.py#L32). Essentially, this is just saying that the Black weight should be considered a regular (non-bold) font. It is then changed to bold in the Android touchup script.